graveler/committed/meta_range_writer.go

```diff
@@ -79,7 +79,12 @@ func (w *GeneralMetaRangeWriter) WriteRecord(record graveler.ValueRecord) error
 	return nil
 }
 
+var ErrNeedBatchClosers = errors.New("need at least 1 batch uploaded")
+
 func (w *GeneralMetaRangeWriter) closeCurrentRange() error {
+	if w.params.MaxUploaders < 1 {
```

Comment:
Why wait until the first range is already written and is being closed to return this error?
Fail it when you first receive this value and create `GeneralMetaRangeWriter`.